public/docs/ts/latest/guide/appmodule.jade

```diff
@@ -0,0 +1,136 @@
+include ../_util-fns
+
+:marked
+  An Angular Module class describes how the application parts fit together.
+  Every application has at least one Angular Module at the _root_ level of the app.
+  You can call it anything you want. The conventional name is `AppModule`.
+
+  Most developers start with a simple `AppModule` and evolve it as the application grows. 
+  Here's a bare-bones `AppModule` from the **QuickStart seed** described on the [setup page](setup.html).
++makeExample('setup/ts/app/app.module.ts','', 'app/app.module.ts')(format='.')

```

Comment:
Nit: drop `(format='.')` since this is a final and complete version of the file?